app/components/Sidebar/components/CollectionLink.js

```diff
@@ -74,49 +78,105 @@ function CollectionLink({
     }),
   });
 
+  //Drop to reorder Collection
+  const [{ isCollectionDropping }, dropToReorderCollection] = useDrop({
+    accept: "collection",
+    drop: async (item, monitor) => {
+      collections.move(
+        item.id,
+        fractionalIndex(collection.index, belowCollectionIndex)
+      );
+    },
+    canDrop: (item, monitor) => {
+      return (
+        collection.id !== item.id &&
+        (!belowCollection || item.id !== belowCollection.id)
+      );
+    },
+    collect: (monitor) => ({
+      isCollectionDropping: monitor.isOver(),
+    }),
+  });
+
+  // Drag to reorder Collection
+  const [{ isCollectionDragging }, dragToReorderCollection] = useDrag({
+    type: "collection",
+    item: {
+      activeCollectionId: ui.activeCollectionId,
+      id: collection.id,
+    },
+    collect: (monitor) => ({
+      isCollectionDragging: monitor.isDragging(),
+    }),
+    canDrag: (monitor) => {
+      return can.move;
+    },
+    begin: (monitor) => {
+      setTimeout(() => {
```

Comment:
I almost forgot why we did this already, needs a comment.

```suggestion
      // setTimeout is required to move this to next render frame otherwise collections
      // below the active collection are not draggable, we believe this is a browser quirk
      setTimeout(() => {
```

Reply:
Actually, not sure this is even working any more as dragging is not collapsing the active collection at all.